src/client/v2/algod/algod.ts

```diff
@@ -43,123 +65,343 @@ export default class AlgodClient extends ServiceClient {
     return new SendRawTransaction(this.c, stxOrStxs);
   }
 
   /**
-   * Returns the given account's information.
+   * Returns the given account's status, balance and spendable amounts.
+   *
+   * #### Example
+   * ```typescript
+   * const address = "XBYLS2E6YI6XXL5BWCAMOA4GTWHXWENZMX5UHXMRNWWUQ7BXCY5WC5TEPA";
+   * const accountInfo = await algodClient.accountInformation(address).do();
+   * ```
+   *
+   * [Response data schema details](https://developer.algorand.org/docs/rest-apis/algod/v2/#get-v2accountsaddress)
    * @param account - The address of the account to look up.
+   * @category GET
    */
   accountInformation(account: string) {
     return new AccountInformation(this.c, this.intDecoding, account);
   }
 
   /**
    * Gets the block info for the given round.
+   *
+   * #### Example
+   * ```typescript
+   * const roundNumber = 18038133;
+   * const block = await algodClient.block(roundNumber).do();
+   * ```
+   *
+   * [Response data schema details](https://developer.algorand.org/docs/rest-apis/algod/v2/#get-v2blocksround)
    * @param roundNumber - The round number of the block to get.
+   * @category GET
    */
   block(roundNumber: number) {
     return new Block(this.c, roundNumber);
   }
 
   /**
    * Returns the transaction information for a specific pending transaction.
+   *
+   * #### Example
+   * ```typescript
+   * const txId = "DRJS6R745A7GFVMXEXWP4TGVDGKW7VILFTA7HC2BR2GRLHNY5CTA";
+   * const pending = await algodClient.pendingTransactionInformation(txId).do();
+   * ```
+   *
+   * [Response data schema details](https://developer.algorand.org/docs/rest-apis/algod/v2/#get-v2transactionspendingtxid)
+   *
+   * @remarks
+   * <br><br>
+   * There are several cases when this might succeed:
+   * - transaction committed (committed round > 0)
+   * - transaction still in the pool (committed round = 0, pool error = "")
+   * - transaction removed from pool due to error (committed round = 0, pool error != "")
+   *
+   * Or the transaction may have happened sufficiently long ago that the node no longer remembers it, and this will return an error.
+   *
    * @param txid - The TxID string of the pending transaction to look up.
+   * @category GET
    */
   pendingTransactionInformation(txid: string) {
     return new PendingTransactionInformation(this.c, txid);
   }
 
   /**
-   * Returns transactions that are pending in the pool.
+   * Returns the list of pending transactions in the pool, sorted by priority, in decreasing order, truncated at the end at MAX.
+   * If MAX = 0, returns all pending transactions.
+   *
+   * #### Example 1
+   * ```typescript
+   * const pendingTxns = await algodClient.pendingTransactionsInformation().do();
+   * ```
+   *
+   * #### Example 2
+   * ```typescript
+   * const maxTxns = 5;
+   * const getPendingTxns = algodClient.pendingTransactionsInformation();
+   * getPendingTxns.max(maxTxns);
+   * const pendingTxns = await getPendingTxns.do();
```

Comment:
nit: a simpler way to do this would be:
```typescript
const pendingTxns = await algodClient
    .pendingTransactionsInformation()
    .max(maxTxns)
    .do();
```

Also applies to `pendingTransactionByAddress`

Reply:
Oh right, done! btw `pendingTransactionByAddress` is returning `PendingTransaction`**s**`ByAddress` 🤔